src/app/analyze/stack/stack-details/stack-details.component.ts

```diff
@@ -1,17 +1,26 @@
 import { Stack } from './../../../models/stack';
```

Comment:
By convention, the first lines of the import are @angular imports, then a space and external libraries, and then internal references.